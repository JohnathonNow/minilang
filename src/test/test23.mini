fun test() do
	for I in 0 .. 10 do
		print('{I + 1} -> ')
		integer::case(I + 1; X) if 2 then
			print('Two')
		elseif 3 then
			print('Three')
		elseif (4, 5) then
			print('Four or five')
		elseif 7 .. 10 then
			print('Seven to ten')
		else
			print('Other')
		end
		print("\n")
	end
end

test()