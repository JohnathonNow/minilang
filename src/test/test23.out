1 -> Other
2 -> Two
3 -> Three
4 -> Four or five
5 -> Four or five
6 -> Other
7 -> Seven to ten
8 -> Seven to ten
9 -> Seven to ten
10 -> Seven to ten
11 -> Other
